Exclude system namespace from admission webhooks in kube prometheus stack (#232)
kyma-project/examples: prometheus/README.md

- Client certificate injection to support scraping of workload secured with Istio strict mTLS
- Active scraping of workload annotated with prometheus.io/scrape

>     >**Note:** If the adminssion webhooks are also being installed then these webhooks should exclude resources being installed in the `kyma-system` namespace. This can be done in the following way:

Reviewer: ```suggestion
    >**Note:** If the admission webhooks are also being installed, then these webhooks must exclude resources being installed in the `kyma-system` Namespace. This can be done in the following way:
```

>     >    -  Add exclusion of `kyma-system` namespace to mutating webhook

Reviewer: ```suggestion
    >    -  Add exclusion of the `kyma-system` Namespace to mutating webhook:
```

    >        ```bash
    >           kubectl -n kyma-system edit mutatingwebhookconfigurations ${HELM_RELEASE}-kube-prometheus-admission
    >        ```

>     >        Exclude `kyma-system` namespace by adding `namespaceSelector`

Reviewer: ```suggestion
    >        Exclude the `kyma-system` Namespace by adding `namespaceSelector`:
```

    >        ```yaml
    >            namespaceSelector:
    >              matchExpressions:
    >              - key: kubernetes.io/metadata.name
    >                operator: NotIn
    >                values:
    >                  - "kyma-system"
    >        ```

>     >   - Add exclusion of `kyma-system` namespace to validationg webhook

Reviewer: ```suggestion
    >   - Add exclusion of the `kyma-system` Namespace to validating webhook:
```

    >        ```bash
    >         kubectl -n kyma-system edit validatingwebhookconfigurations ${HELM_RELEASE}-kube-prometheus-admission
    >        ```

>     >        Exclude `kyma-system` namespace by adding `namespaceSelector`

Reviewer: ```suggestion
    >        Exclude the `kyma-system` Namespace by adding `namespaceSelector`:
```
